packages/design-system-web/src/components/geov-authority-lookup-explorer/lib/const.tsx

```diff
@@ -39,3 +39,26 @@ export const TYPE_ALL_IDREF = 'all:';
 export const TYPE_PERSON_IDREF = 'persname_t:';
 export const TYPE_PLACE_IDREF = 'geogname_t:';
 export const TYPE_GROUP_IDREF = 'corpname_t:';
+
+// Geovistory
+export const TITLE_GEOVISTORY = 'Geovistory';
+export const ENDPOINT_SPARQL_GEOVISTORY = 'https://sparql.geovistory.org/api_v1_community_data';
+export const QR_SPARQL_GEOVISTORY = (keywords: string, type: string, pageSize: number) => `
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX text: <http://jena.apache.org/text#>
+PREFIX ontome: <https://ontome.net/ontology/>
+
+SELECT DISTINCT ?item ?name ?classUri
+WHERE
+{
+  	(?item) text:query ('${keywords}') .
+  	?item rdfs:label ?name .
+  	?item a ?classUri .
+  	${type}
+}
+LIMIT ${pageSize}
+OFFSET 0`;
+
+export const TYPE_PERSON_GEOVISTORY = 'FILTER (?classUri = ontome:c21)';
```

Comment:
why do you use a filter and not directly:

```
?item a ontome:c21
```

Sparql filter has to loop over rows while the other approach uses indexes and is much faster.

The overall solution could be:
```sparql
SELECT DISTINCT ?item ?name ?classUri
WHERE
{
  	(?item) text:query ('${keywords}') .
  	?item rdfs:label ?name .
  	?item a ${type} . 
}
LIMIT ${pageSize}
OFFSET 0`;

export const TYPE_PERSON_GEOVISTORY = 'ontome:c21';
...
```

Reply:
We need ?classUri and we need an usecase for no choice
But you're right, we can avoid FILTER(): ?item a ?classUri, ontome:c21 . 
Done